Library/Homebrew/cmd/bottle.rb

```diff
@@ -183,6 +186,10 @@ def bottle_formula(f)
     relocatable = false
     skip_relocation = false
 
+    formula_last_commit_hash, formula_last_commit_time = \
+      *Utils.popen_read("git", "log", "--pretty=%H%cd", "--", f.path).split(" ")
+    formula_last_commit_time = Time.parse(formula_last_commit_time)
```

Comment:
As explained below, I think such logic should be more suitable to put in formula installer. While, here, we can read the time directly from tab file.